### PR TITLE
QoL improvements for augmented people

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
balance: People with augmented libs can now heal them with a welder or cable coils instantly much like fleshy people can with brute packs/ointment/patches
balance: The chaplains bible can now heal augmented people, however it only heals brute or burn and only heals one limb at a time. Keep some mannitol handy. 
/:cl:

[why]: When we changed patches/ointment/brute packs to be instant I forgot that tg nerfed augmented healing too. Augmented healing only heals 15 per use of cable/welder whereas ointment/brute packs heal 40. I changed the bible too because sometimes chaplains run a Tech-priest meme and its sad they cannot naturally heal their converts with the power of the Omnissiah.

### CODE REVIEW

**Comment by PopNotes** [comment]
Gud +1

**Comment by ghost** [comment]
oil should give augments ephedrine speed but confusion due to higher viscosity and low friction

File: hippiestation/code/game/objects/items/storage/book.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+/obj/item/storage/book/bible/bless(mob/living/carbon/human/H, mob/living/user)
+	var/heal_amt = 10
+	var/list/hurt_limbs = H.get_damaged_bodyparts(1, 1)
+
+	if(hurt_limbs.len)
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/storage/book.dm]: LAZYLEN

File: hippiestation/code/game/objects/items/storage/book.dm [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		for(var/X in hurt_limbs)
+			var/obj/item/bodypart/affecting = X
+			if(affecting.status == BODYPART_ROBOTIC)
+				if(item_heal_robotic(H, user, heal_amt ,0))//You may only heal one bodypart at a time using this proc so you do have to hit them more often.
+					H.visible_message("<span class='notice'>[user] mends the dents of [H] with the power of [deity_name]!</span>")
+					to_chat(H, "<span class='boldnotice'>May the power of [deity_name] find your metal unwanting!</span>")
+					playsound(src.loc, 'sound/items/drill_use.ogg', 25, 1, -1)
+					return 1
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/storage/book.dm]: TRUE

File: hippiestation/code/game/objects/items/storage/book.dm [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+				else if(item_heal_robotic(H, user, 0, heal_amt ))
+					H.visible_message("<span class='notice'>[user] fixes the wire of [H] with the power of [deity_name]!</span>")
+					to_chat(H, "<span class='boldnotice'>May the power of [deity_name] restore your currents!</span>")
+					playsound(src.loc, 'sound/items/drill_use.ogg', 25, 1, -1)
+					return 1
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/storage/book.dm]: TRUE

File: hippiestation/code/game/objects/items/storage/book.dm [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+		for(var/X in hurt_limbs)
+			var/obj/item/bodypart/affecting = X
+			if(affecting.status == BODYPART_ROBOTIC)
+				if(item_heal_robotic(H, user, heal_amt ,0))//You may only heal one bodypart at a time using this proc so you do have to hit them more often.
+					H.visible_message("<span class='notice'>[user] mends the dents of [H] with the power of [deity_name]!</span>")
+					to_chat(H, "<span class='boldnotice'>May the power of [deity_name] find your metal unwanting!</span>")
+					playsound(src.loc, 'sound/items/drill_use.ogg', 25, 1, -1)
+					return 1
+				else if(item_heal_robotic(H, user, 0, heal_amt ))
+					H.visible_message("<span class='notice'>[user] fixes the wire of [H] with the power of [deity_name]!</span>")
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/storage/book.dm]: wires

File: hippiestation/code/game/objects/items/storage/book.dm [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+					to_chat(H, "<span class='boldnotice'>May the power of [deity_name] restore your currents!</span>")
+					playsound(src.loc, 'sound/items/drill_use.ogg', 25, 1, -1)
+					return 1
+			else if(affecting.heal_damage(heal_amt, heal_amt))
+				H.update_damage_overlays()
+				H.visible_message("<span class='notice'>[user] heals [H] with the power of [deity_name]!</span>")
+				playsound(src.loc, "punch", 25, 1, -1)
+				return 1
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/storage/book.dm]: TRUE

File: hippiestation/code/game/objects/items/tools.dm [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+	if(!istype(H))
+		return ..()
+
+	var/obj/item/bodypart/affecting = H.get_bodypart(check_zone(user.zone_selected))
+
+	if(affecting && affecting.status == BODYPART_ROBOTIC && user.a_intent != INTENT_HARM)
+		if(src.remove_fuel(1))
+			playsound(loc, usesound, 50, 1)
+			if(user == H)
+				user.visible_message("<span class='notice'>[user] starts to fix some of the dents on [H]'s [affecting.name].</span>", "<span class='notice'>You start fixing some of the dents on [H]'s [affecting.name].</span>")
```
Comment (ghost) [inline hippiestation/code/game/objects/items/tools.dm]: Why is it it "starts" if its immediate?

File: hippiestation/code/modules/power/cable.dm [hunk 7]
```diff
@@ -1,3 +1,18 @@
...
+
+
+/obj/item/stack/cable_coil/attack(mob/living/carbon/human/H, mob/user)
+	if(!istype(H))
+		return ..()
+
+	var/obj/item/bodypart/affecting = H.get_bodypart(check_zone(user.zone_selected))
+	if(affecting && affecting.status == BODYPART_ROBOTIC)
+		if(user == H)
+			user.visible_message("<span class='notice'>[user] starts to fix some of the wires in [H]'s [affecting.name].</span>", "<span class='notice'>You start fixing some of the wires in [H]'s [affecting.name].</span>")
```
Comment (ghost) [inline hippiestation/code/modules/power/cable.dm]: Why is it it "starts" if its immediate?

**Review by ghost — DISMISSED** [review]

File: hippiestation/code/modules/power/cable.dm [hunk 8]
```diff
@@ -1,3 +1,18 @@
...
+			if(item_heal_robotic(H, user, 0, 15))
+				use(1)
+			return
```
Comment (KayeArray) [inline hippiestation/code/modules/power/cable.dm]: you put a return here but not in the other code

File: hippiestation/code/modules/power/cable.dm (hunk 7 above)
Comment (GuyonBroadway) [reply hippiestation/code/modules/power/cable.dm]: Holdover after gutting the time delay

File: hippiestation/code/game/objects/items/tools.dm (hunk 6 above)
Comment (GuyonBroadway) [reply hippiestation/code/game/objects/items/tools.dm]: Holdover after gutting the time delay.

**Review by KayeArray — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/power/cable.dm (hunk 8 above)
Comment (GuyonBroadway) [reply hippiestation/code/modules/power/cable.dm]: welders have an afterattack and other stuff.

Comment (GuyonBroadway) [reply hippiestation/code/modules/power/cable.dm]: Not to mention this is copy pasted from how they currently work.

**Review by KayeArray — APPROVED** [review]

**Comment by KayeArray** [comment]
if this is tested I see no problem +1

**Comment by GuyonBroadway** [comment]
Yup its all tested and kosher.

**Comment by DinduStuffin** [comment]
+1
The only reason I'm never using augs is because of the delayed healing garbage.

**Comment by PopNotes** [comment]
While I do agree with this, it just might return us to the age of everyone breaking into robo demanding to be Jensen'd

**Comment by ghost** [comment]
good change

**Comment by ghost** [comment]
sure

**Comment by PopNotes** [comment]
Doesn't this only have +1

**Pull request merged**